### DISCUSSION TITLE
Home Assistant Warnings about Phase angle

### DISCUSSION THREAD

**Discussion by copekyle** [emporia-vue-local/esphome#122] [Q&A]
I'm not sure if this is the right place for it but the new home assistant core is throwing an error about the Phase angle frequency not being a valid unit. Here is the error:

Logger: homeassistant.components.sensor
Source: components/sensor/__init__.py:677
Integration: Sensor (documentation, issues)
First occurred: 10:50:25 AM (2 occurrences)
Last logged: 10:50:25 AM

Entity sensor.phase_a_phase_angle (<class 'homeassistant.components.esphome.sensor.EsphomeSensor'>) is using native unit of measurement '°' which is not a valid unit for the device class ('frequency') it is using; expected one of ['GHz', 'kHz', 'Hz', 'MHz']; Please update your configuration if your entity is manually configured, otherwise create a bug report at https://github.com/home-assistant/core/issues?q=is%3Aopen+is%3Aissue+label%3A%22integration%3A+esphome%22

Entity sensor.phase_b_phase_angle (<class 'homeassistant.components.esphome.sensor.EsphomeSensor'>) is using native unit of measurement '°' which is not a valid unit for the device class ('frequency') it is using; expected one of ['GHz', 'kHz', 'Hz', 'MHz']; Please update your configuration if your entity is manually configured, otherwise create a bug report at https://github.com/home-assistant/core/issues?q=is%3Aopen+is%3Aissue+label%3A%22integration%3A+esphome%22

I know it says to create a bug elsewhere but I see the point in the error. Frequency is technically in hertz and not degrees.

**Comment by flaviut** [comment]
Should be fixed with https://github.com/emporia-vue-local/esphome/commit/4dfad0f22bf103e7e165185c4df968a664e4854b, thanks for pointing it out :)

**Reply by CommittotheIndian** [reply]
I'm running into a similar issue when trying to add Phase Angle and Phase Frequency. ESPhome will not validate. I think you provided the solution above but I can not find that sensor anywhere in the files to update or change.

**Reply by flaviut** [reply]
Seems like a different issue, please make another post & include the relevant part of the logs & config.
